_posts/2022-09-10-this-week-in-rails-rack-3-and-rails-7-0-4-6-1-7-and-6-0-6-releases-actiondispatch-cookies-etc-67ee6b56.markdown

```diff
@@ -2,7 +2,7 @@
 layout: post
 title: "Rack 3 and Rails 7.0.4, 6.1.7, and 6.0.6 releases, ActionDispatch::Cookies, etc"
 categories: news
-author: CHANGELOG.md
+author: siaw23
```

Comment:
Don't you want to put your real name here?

Reply:
Nope, it's username for everyone else, so I'm doing the same.